app/controllers/api/v2/courses_controller.rb

```diff
@@ -16,7 +16,20 @@ def show
       end
 
       def sync_with_search_and_compare
+        response = ManageCoursesAPIService::Request.sync_course_with_search_and_compare(
+          @current_user.email,
+          @provider.provider_code,
+          @course.course_code
+        )
+
+        head response ? :ok : :internal_server_error
```

Comment:
For the future ... this is going to swallow up errors and not log them. We should not do that.

Reply:
Added a note to expose errors to the ticket about syncing courses (which will re-write this part).